static/org-settings.html

```diff
@@ -62,10 +62,40 @@
                                 <th>Retention</th>
                                 <td id="retention-days-value">4 days</td>
                             </tr>
+                            <tr>
+                                <th>Persistent Queries</th>
+                                <td>
+                                    <div class="dropdown">
+                                        <button class="btn dropdown-toggle" type="button" id="contact-types" style=" border-radius: 0; padding: 0.1rem 0.1rem "
+                                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
+                                            data-bs-toggle="dropdown">
+                                            <span style="color: var(--health-stats-value-text-color);font-size: 18px;">Enabled</span>
```

Comment:
I think we recently changed the default font size. 18 is too large but it looks good if you just remove `font-size: 18px;`. I'm not sure if there's a different way you should change it to make sure it's consistent with the rest of the fonts.